python/link_prediction/tests/processing_tests.py

```diff
@@ -0,0 +1,34 @@
+import mgp
```

Comment:
Since this is deep into implementation, this file shouldn't have reference to `mgp`. Especially because in the implementation we use DGS graph representation, which is independent of Memgraph. Suggest you implement this method on main script level.